src/components/MessagesList/MessagesList.vue

```diff
@@ -50,6 +50,13 @@ get the messagesList array and loop through the list to generate the messages.
 				v-for="n in 15"
 				:key="n" />
 		</template>
+		<transition name="fade">
+			<button v-show="!isScrolledToBottom"
+				class="scroll-to-bottom"
+				@click="scrollToBottom">
+				<ChevronDown :size="24" fill-color="#fff" />
```

Comment:
only works when the color on primary is "white"

Reply:
We're going to have a design hack Tuesday next week and by the end of that we'll have a way to use material design icons with theming

Reply:
`var(--color-primary-text)` it is !